tl/candidate_generation/utility.py

```diff
@@ -168,5 +151,5 @@ def write_auxiliary_files(self, auxiliary_folder, all_candidates_aux_dict, auxil
 
             for key in _:
                 df = pd.DataFrame(_[key])
-                if len(df) > 0:
```

Comment:
why did you remove this check?

Reply:
Actually, this check was not there in the master branch, which I had cloned for my experiments. So it did not show up in my fork. Should I add it to the function?

Reply:
you should sync your fork with `development` first and then create this pull request. I will close this PR